nixos/modules/services/torrent/transmission.nix

```diff
@@ -264,13 +264,27 @@ in
     # because BindPaths= needs these directories before.
     system.activationScripts = mkIf (cfg.downloadDirPermissions != null)
       { transmission-daemon = ''
-        install -d -m 700 '${cfg.home}/${settingsDir}'
-        chown -R '${cfg.user}:${cfg.group}' ${cfg.home}/${settingsDir}
-        install -d -m '${cfg.downloadDirPermissions}' -o '${cfg.user}' -g '${cfg.group}' '${cfg.settings.download-dir}'
+        install -d -m 700 ${escapeShellArg cfg.home}/${settingsDir}
+        chown -R \
+          ${escapeShellArg cfg.user}:${escapeShellArg cfg.group} \
+          ${escapeShellArg cfg.home}/${settingsDir}
+        install -d \
+          -m ${escapeShellArg cfg.downloadDirPermissions} \
+          -o ${escapeShellArg cfg.user} \
+          -g ${escapeShellArg cfg.group} \
+          ${escapeShellArg cfg.settings.download-dir}
         '' + optionalString cfg.settings.incomplete-dir-enabled ''
-        install -d -m '${cfg.downloadDirPermissions}' -o '${cfg.user}' -g '${cfg.group}' '${cfg.settings.incomplete-dir}'
+        install -d \
+          -m ${escapeShellArg cfg.downloadDirPermissions} \
+          -o ${escapeShellArg cfg.user} \
+          -g ${escapeShellArg cfg.group} \
+          ${escapeShellArg cfg.settings.incomplete-dir}
         '' + optionalString cfg.settings.watch-dir-enabled ''
-        install -d -m '${cfg.downloadDirPermissions}' -o '${cfg.user}' -g '${cfg.group}' '${cfg.settings.watch-dir}'
+        install -d \
+          -m ${escapeShellArg cfg.downloadDirPermissions} \
+          -o ${escapeShellArg cfg.user} \
+          -g ${escapeShellArg cfg.group} \
+          ${escapeShellArg cfg.settings.watch-dir}
```

Comment:
Could we separate formatting changes such as this to a separate commit? It'd make it easier to review this PR.